lightning/src/util/config.rs

```diff
@@ -206,7 +206,21 @@ pub struct ChannelConfig {
 	/// This cannot be changed after a channel has been initialized.
 	///
 	/// Default value: true.
-	pub commit_upfront_shutdown_pubkey: bool
+	pub commit_upfront_shutdown_pubkey: bool,
+	/// Limit holder balance exposure to dusted HTLCs at any given time.
+	///
+	/// Applied on outgoing dust HTLCs and incoming dust HTLCs committed on the counterparty
+	/// commitment transaction.
+	///
+	/// Dust HTLCs are defined as any HTLC lower than counterparty's `dust_limit_satoshis` summed
```

Comment:
Users have no idea what `dust_limit_satoshis` is, or why it matters at all what the sum of dust HTLCs is. We need to say something that describes, without assuming any significant knowledge of lightning, *why* we limit the dust balance to some low value (that those funds go to miners, and that a malicious miner could forward lots of small payments through us that miners claim as additional fee revenue if we force-close).